trinity/utils/validation.py

```diff
@@ -30,3 +34,11 @@ def validate_transaction_call_dict(transaction_dict: Dict[str, Any], vm: BaseVM)
                 list(sorted(spec_keys)),
             )
         )
+
+
+def validate_transaction_call_dict(transaction_dict: Dict[str, Any], vm: BaseVM) -> None:
+    validate_transaction_gas_estimation_dict(transaction_dict, vm)
+
+    # 'to' is required in a call, but not a gas estimation
+    if not is_address(transaction_dict.get('to', '')):
```

Comment:
*nitpick* Do we need the default empty string here?  Wouldn't the default return of `None` also work?